apps/frontend/app/components/flows/AddExperiment/stepComponents/DispatchStep.tsx

```diff
@@ -1,7 +1,7 @@
 'use client'
 
 import { Dropzone, DropzoneProps } from '@mantine/dropzone';
-import { submitExperiment, uploadExec } from '../../../../../firebase/db';
+import { submitExperiment } from '../../../../../firebase/db';
```

Comment:
can we change the file structure naming?

Reply:
I will change this when AuthJS is implemented.